src/transformers/pipelines/base.py

```diff
@@ -1035,12 +856,27 @@ def forward(self, model_inputs, **forward_params):
                     model_inputs = self._ensure_tensor_on_device(model_inputs, device=self.device)
                     model_outputs = self._forward(model_inputs, **forward_params)
                     model_outputs = self._ensure_tensor_on_device(model_outputs, device=torch.device("cpu"))
+            elif self.framework == "flax":
+                from jax.numpy import DeviceArray
+
+                model_outputs = self._forward(model_inputs, **forward_params)
+                # Back to numpy
+                model_outputs = model_outputs.__class__(
+                    **{k: v.to_py() if isinstance(v, DeviceArray) else v for k, v in model_outputs.items()}
+                )
             else:
                 raise ValueError(f"Framework {self.framework} is not supported")
         return model_outputs
 
     def get_iterator(
-        self, inputs, num_workers: int, batch_size: int, preprocess_params, forward_params, postprocess_params
+        self,
+        inputs,
+        num_workers: int,
+        batch_size: int,
+        preprocess_params,
+        forward_params,
+        postprocess_params,
+        align_to: Optional[int] = None,
```

Comment:
Does this mean it's "bucketed" to 8 or padded to 8? Not a huge fan of `align_to` - would prefer `bucket_to_multiple_of`

Reply:
also maybe make this private in a first iteration to clearly state that it's experimental?

Reply:
It's not bucketing, it is forcing final `seq_length` to a specific mutliple of `align_to`.

Is `pad_to_multiple_of` more understandable to you ?